addons/account/tests/test_duplicate_res_partner_bank.py

```diff
@@ -49,3 +49,7 @@ def test_remove_bank_account_from_partner(self):
             partner_form.bank_ids.remove(0)
 
         self.assertEqual(len(self.partner_a.bank_ids), 0)
+
+    def test_duplicate_acc_number_inactive_bank_account(self):
+        self.partner_bank_b.active = False
+        self.assertFalse(self.partner_bank_a.duplicate_bank_partner_ids, self.partner_a)
```

Comment:
I don't think we need self.partner_a in the message, makes it less clear what the issue is if the test fails, what do you think?

```suggestion
        self.assertFalse(self.partner_bank_a.duplicate_bank_partner_ids)
```